torch/jit/__init__.py

```diff
@@ -1250,6 +1250,9 @@ def forward(self, input):
         if not _is_new_style_class(obj):
             raise RuntimeError("TorchScript classes must be new-style classes. "
                                "Please inherit from 'object'")
+        if len(obj.mro()) > 2:
+            raise RuntimeError("TorchScript classes does not support inheritance or any polymorphism strategy yet. "
```

Comment:
"or any polymorphism strategy" is not correct. Interfaces are a form of polymorphism.  The first part of the message is fine.